modules/machineset-yaml-rhv.adoc

```diff
@@ -84,8 +83,6 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <6> Specify the UUID for the {rh-virtualization} cluster to which this VM instance belongs.
 
 <7> Specify the {rh-virtualization} VM template to use to create the machine.
-
-<8> Optional: Specify the VM instance type.
 +
 [WARNING]
```

Comment:
This warning regarding the deprecation of the `instance_type_id` should be removed as well.